qcodes/instrument_drivers/stanford_research/SR830.py

```diff
@@ -442,6 +442,14 @@ def parse_offset_get(s: str) -> Tuple[float, int]:
                            get_parser=float,
                            unit='deg')
 
+        self.add_parameter(
+            "complex_voltage",
+            label="Voltage",
+            get_cmd=self._get_complex_voltage,
+            unit="V",
+            vals=ComplexNumbers(),
```

Comment:
Should this have a docstring. Especially with respect to x,y or mag + phase

Reply:
Good point, I'll add in a doc string.